oops/observation/__init__.py

```diff
@@ -955,6 +951,122 @@ def inventory(self, bodies, tfrac=0.5, time=None, expand=0.,
         raise NotImplementedError(type(self).__name__ + '.inventory '
                                   'is not implemented')
 
+    ############################################################################
+    # Support for parallel observations
```

Comment:
I find the word "parallel" confusing in this context because it has a geometric connotation implying that the lines of sight are the same.   I would pfrefer "coupled" instead.

"Simultaneous" or "concurrent" are also ok, but they really only imply the same time, not the same place ( and "simultaneous" doen't even imply the same time in relativity).

Reply:
The term "parallel" is used throughout HST and JWST documentation, so I guess that is where I picked it up. I don't think the other terms you propose really capture the correct meaning. The requirements are same time, same observing platform, and using instruments that cannot be pointed independently of one another. Cassini ISS BOTSIMs and ORS riders would qualify as parallels. Let's discuss.

Reply:
Sounds good.  It's difficult to find the exact right word.

Reply:
I think you're both being too literal. For all practical purposes, they _are_ parallel lines of sight--multiple instruments looking at the same target at the same time from the same host, using instruments that are designed to be used in (dare I say it) parallel.